packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -24,21 +27,45 @@ class LinkData with SearchableDataMixin {
     this.pathError = false,
   });
 
-  final String path;
-  final String domain;
+  final List<String> path;
```

Comment:
I don't think we should make them list, same for domain. It will create different representation for a same set of data. if we want to group by we can do that on the fly or create a different data class for that.